templates/etc.postfix.main.cf.j2

```diff
@@ -31,7 +31,7 @@ mynetworks = {{ postfix_mynetworks | join(', ') }}
 {% endif %}
 
 {% if postfix_relayhost %}
-relayhost = [{{ postfix_relayhost }}]
```

Comment:
This format is intentional:

> The form [hostname] turns off MX lookups.

Is there a reason you're changing it?

Reply:
I needed to add the port number in the relayhost variable since I couldn't use the default SMTP port. So removing the brackets allowed me to define the relayhost variable like this 

`postfix_relayhost: "[email-smtp.us-east-1.amazonaws.com]:587"`  

The other solution would be to add an additional variable for port.

Reply:
add an additional var

Reply:
will do.